### PR TITLE
[WIP] Replace LegacyWaffleFlag usage with WaffleFlag

### PR DESCRIPTION
**Pull request by dyudyunov**
Closes https://github.com/openedx/public-engineering/issues/28

YT: https://youtrack.raccoongang.com/issue/OeX_Depr-86

### CODE REVIEW

File: openedx/core/djangoapps/schedules/config.py [hunk 1]
```diff
@@ -3,12 +3,12 @@
...
 from crum import get_current_request
-from edx_toggles.toggles import LegacyWaffleSwitch, LegacyWaffleFlagNamespace, LegacyWaffleSwitchNamespace, WaffleFlag
+from edx_toggles.toggles import LegacyWaffleSwitch, LegacyWaffleSwitchNamespace, WaffleFlag
 
 from lms.djangoapps.experiments.flags import ExperimentWaffleFlag
 from lms.djangoapps.experiments.models import ExperimentData
 
-WAFFLE_FLAG_NAMESPACE = LegacyWaffleFlagNamespace(name='schedules')
+WAFFLE_FLAG_NAMESPACE = 'schedules'
 WAFFLE_SWITCH_NAMESPACE = LegacyWaffleSwitchNamespace(name='schedules')
```
Comment (ruzniaievdm) [inline openedx/core/djangoapps/schedules/config.py:12]: you can pass the `WAFFLE_FLAG_NAMESPACE` to `WAFFLE_SWITCH_NAMESPACE = LegacyWaffleSwitchNamespace(name=WAFFLE_FLAG_NAMESPACE)`

Comment (dyudyunov) [reply openedx/core/djangoapps/schedules/config.py:12]: switches stay as they were, all corresponding changes will be provided in the following commits

**Comment by dyudyunov** [comment]
> Some minor nits about double and single quotes, you use them together Perhaps it is worth following to one standard?

Ops, I did it again... Thanks, I'll fix that

File: cms/djangoapps/models/settings/waffle.py [hunk 2]
```diff
@@ -3,15 +3,14 @@
...
 WAFFLE_NAMESPACE = 'grades'
 
 # edx/edx-platform feature
 MATERIAL_RECOMPUTE_ONLY = 'MATERIAL_RECOMPUTE_ONLY'
 
 MATERIAL_RECOMPUTE_ONLY_FLAG = CourseWaffleFlag(  # lint-amnesty, pylint: disable=toggle-missing-annotation
-    waffle_namespace=WAFFLE_NAMESPACE,
-    flag_name=MATERIAL_RECOMPUTE_ONLY,
-    module_name=__name__,
+    f'{WAFFLE_NAMESPACE}.{MATERIAL_RECOMPUTE_ONLY}', __name__
```
Comment (idegtiarov) [inline cms/djangoapps/models/settings/waffle.py:13]: We do not have a prefix here, should we add it?

**Review by idegtiarov — COMMENTED** [review]
One question is inline, other LGTM

File: cms/djangoapps/models/settings/waffle.py (hunk 2 above)
Comment (dyudyunov) [reply cms/djangoapps/models/settings/waffle.py:13]: It's not necessary and I tried not to change the initial flow

**Review by idegtiarov — APPROVED** [review]
Thank for the new commit, it LGTM! Waiting for the commit with the test updates.

**Comment by dyudyunov** [comment]
All filed tests can't be fixed in this PR, they require modifications for ORA repo to use modern Waffles

**Comment by dyudyunov** [comment]
Pipelines fail because there is no required edx-toggles and ORA versions yet

**Review by idegtiarov — APPROVED** [review]
The changes with the tests updates and version upgrade LGTM!

**Review by ruzniaievdm — APPROVED** [review]

**Pull request closed without merging**